src/apps/api/tests/test_queue_api_permissions.py

```diff
@@ -104,3 +99,42 @@ def test_organizer_user_cannot_delete_or_edit_queue(self):
         )
         assert response.status_code != 204
         assert Queue.objects.filter(pk=self.created_queue.id).exists()
+
+    # Test user queue limits
+    def test_queue_limits_for_regular_users(self):
+        [QueueFactory(owner=self.user) for _ in range(3)]
+
+        self.client.login(username='test', password='test')
+        data = self._get_test_update_data(return_id=False)
+        # default limit is 5
+        response = self.client.post(
+            reverse('queues-list'),
+            data=data,
+            content_type='application/json'
+        )
+        assert response.status_code == 201
+        # Next one should fail!
+        response = self.client.post(
+            reverse('queues-list'),
+            data=data,
+            content_type='application/json'
+        )
+        assert response.status_code == 400
+
+    # Test user queue limits
+    def test_queue_limits_for_super_users(self):
```

Comment:
rename test to something like
`test_super_users_have_no_queue_limit`
since that is what we are actually doing.

Reply:
✔️